PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +609,45 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (PluginBase.CurrentProject == null || 
```

Comment:
I think we don't need to limit to AS3/Haxe - we don't have this condition when looking up /* ... */ comments as we don't even have the concept of code VS non-code files.